.github/workflows/render-all.yml

```diff
@@ -63,21 +63,6 @@ jobs:
         run: |
           Rscript -e "bookdown::render_book('index.Rmd', output_format = 'all');
           file.copy(from = 'assets', to = 'docs/assets', overwrite = TRUE)"
```

Comment:
```suggestion
          file.copy(from = 'assets', to = 'docs/assets', overwrite = TRUE)"
          
      # Render a student guide if specified. This is a bit clunky because 
      # Bookdown does not work well if the files aren't named as such in the 
      # root directory
      - name: Run student guide render
        if: ${{needs.yaml-check.outputs.render_student_guide == 'yes'}}
        id: student_guide
        run: |
          mkdir tmp1
          mv _bookdown.yml _output.yml tmp1
          mv student-guide/_bookdown.yml student-guide/_output.yml .
          Rscript -e "bookdown::render_book('index.Rmd', output_format = 'all')"
          mv _bookdown.yml _output.yml student-guide
          mv tmp1/_bookdown.yml tmp1/_output.yml .
          rm -r tmp1
```